base/abstractarray.jl

```diff
@@ -570,10 +570,10 @@ to_shape(r::OneTo) = Int(last(r))
 to_shape(r::AbstractUnitRange) = r
 
 """
-    similar(storagetype, indices)
+    similar(storagetype, axes)
```

Comment:
This one could have been kept, but it's more consistent with the (not visible) example below, which passes `axes(A)` as this argument.